packages/OptionButton/src/option-button.scss

```diff
@@ -1,157 +1,265 @@
-  --ids-option-button-text-color: #{tokens.$grey-800};
-  --ids-option-button-text-font-size: #{tokens.$font-size-5};
-  --ids-option-button-text-font-weight: #{tokens.$font-weight-semi-bold};
-  --ids-option-button-text-line-height: #{tokens.$line-height-xl};
-
-  /* Description */
-  --ids-option-button-desc-color: #{tokens.$grey-600};
-  --ids-option-button-desc-font-size: #{tokens.$font-size-3};
-  --ids-option-button-desc-line-height: #{tokens.$line-height-xxl};
+    /* Default */
+    --ids-option-button-border-color: #{tokens.$grey-400};
+    --ids-option-button-border-dimension: 0.1rem;
+    --ids-option-button-border-radius: #{tokens.$border-radius-sm};
+    --ids-option-button-font-family: #{tokens.$primary-font-family};
+    --ids-option-focus: 0 0 0 0.2rem #ffffff, 0 0 0 0.35rem #0C46D0;
+    --ids-option-button-padding: 0;
+
+    /* Hover */
+    --ids-option-button-hover-border-color: #{tokens.$grey-600};
+    --ids-option-button-hover-background-color: transparent;
+
+    /* Checked */
+    --ids-option-button-checked-box-shadow: 0 0 0.6rem #3c7dff;
+    --ids-option-button-checked-border-color: transparent;
+    --ids-option-button-checked-background-color: transparent;
+    --ids-option-button-checked-outline: none;
+    --ids-option-button-checked-outline-offset: 0;
+
+    /* Disabled */
+    --ids-option-button-icon-container-disabled-background: #{tokens.$grey-200};
+    --ids-option-button-border-color-disabled: #{tokens.$grey-400};
+    --ids-option-button-background-color-disabled: transparent;
+
+    /* Icon */
+    --ids-option-button-icon-container-background: #{tokens.$electric-blue-50};
+    --ids-option-button-icon-container-height: 5.6rem;
+    --ids-option-button-icon-container-width: 6.2rem;
+    --ids-option-button-icon-background-disabled: #{tokens.$grey-300};
+    --ids-option-button-icon-width: auto;
+    --ids-option-button-icon-height: auto;
+    --ids-option-button-icon-circle-fill: #{tokens.$sky-100};
+    --ids-option-button-icon-path-fill: #{tokens.$grey-800};
+    --ids-option-button-icon-stroke: #{tokens.$grey-800};
+    --ids-option-button-icon-circle-fill-hover: #{tokens.$sky-100};
+    --ids-option-button-icon-circle-fill-disabled: #{tokens.$sky-100};
+
+    /* Text */
+    --ids-option-button-text-container-padding: #{tokens.$space-1} #{tokens.$space-3};
+    --ids-option-button-text-color: #{tokens.$grey-800};
+    --ids-option-button-text-font-size: #{tokens.$font-size-5};
+    --ids-option-button-text-font-weight: #{tokens.$font-weight-semi-bold};
+    --ids-option-button-text-line-height: #{tokens.$line-height-xl};
+
+    /* Description */
+    --ids-option-button-desc-color: #{tokens.$grey-600};
+    --ids-option-button-desc-font-size: #{tokens.$font-size-3};
+    --ids-option-button-desc-line-height: #{tokens.$line-height-xxl};
+}
+
+[data-brand="workleap"] {
+    /* Default */
+    --ids-option-button-padding: 0 0 0 16px;
```

Comment:
the `px` value seems to be forgotten, and it's possible to use a token for this -> `--hop-space-inline-md`

Reply:
Totally!